ProjFS.Mac/PrjFSKextTests/MockProc.cpp

```diff
@@ -0,0 +1,23 @@
+#include "MockProc.hpp"
+#include <string.h>
+
+int proc_pid(proc_t)
+{
+    return 1;
+}
+
+char* return_proc_name;
```

Comment:
I don't think we need this variable outside this compilation unit, so it can be `static`. I'm not sure what our naming convention for global variables is, but for file-level `static` variables it's `s_camelCase`.